### ISSUE TITLE
[MAS4.1.2][Screen reader-Welcome] Screen reader is not reading the state for Welcome tab control when the user selected.

### ISSUE THREAD

**Issue by awalia13** [microsoft/BotFramework-Emulator#1717]
**Actual Result:**
Screen reader is not reading the state for Welcome tab control when the user selected which is present in the welcome tab screen 

**Expected Result:**
Screen reader should narrate the state for Welcome tab control when the user 
Example: Screen reader should narrate as Welcome tab is selected when the user selected the welcome tab screen 

**Note:** Screen reader is not reading the selected state for when  the user selected the user message text.
Please find the attachment:MAS4.1.2_Live.chat.zip

Note: Same issue repros in NVDA also. Please refer the attachment as <MAS4.1.2_Tab controls_NVDA.wmv>

**Repro Steps:**
Turn on the narrator 
1. Open the Bot Framework V4 Emulator.
2. Navigate to the Welcome tab screen 
3. Select the Welcome tab  
4. Observe the issue

**User Impact:**
Users who rely on screen reader will be affected to navigate the control if screen reader is not reading the state then user will  be affected which tab is selected.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Screen reader: Narrator

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
](url)
 
[MAS4.1.2_Welcome.zip](https://github.com/microsoft/BotFramework-Emulator/files/3491785/MAS4.1.2_Welcome.zip)
![MAS4 1 2_Welcome](https://user-images.githubusercontent.com/46575396/62857524-7ec03780-bd15-11e9-839b-6071a9578cd8.png)
[MAS4.1.2_Live chat.zip](https://github.com/microsoft/BotFramework-Emulator/files/3525114/MAS4.1.2_Live.chat.zip)

[MAS4.1.2_Tab controls_NVDA.zip](https://github.com/microsoft/BotFramework-Emulator/files/3546027/MAS4.1.2_Tab.controls_NVDA.zip)

**Comment by tonyanziano** [comment]
@awalia13 @Kumar2608 

I'm confused about what the issue is here. The bug title, description, and comments are referring to 2 completely different bugs.

1. The tab control is not reading the "selected" state. I have tested this with the latest Emulator, and I cannot reproduce the issue. It is reading correctly:

> "Tab control, welcome tab 1 of 2"

and when navigating to the settings tab that is selected:

> "Emulator settings tab, 2 of 2, selected"

on select: 

> "Selected"

2. The second bug is referring to the selected state of messages in the chat panel. This should be a separate bug as it is not related at all to the tab control.

**Comment by tonyanziano** [comment]
After looking into the issue further, the reason it was reopened was because the selected state of the user message needs to be narrated as well. It was unclear that this was also part of the original bug.

**Comment by awalia13** [comment]
Hi Tony, if same type of issue repro on another control then we generally log the single bug and mention it in Note section. This is the common practice we follow. Now the main issue has fixed but the issue mentioned in Note Section is not fixed. 
Sorry for the confusion.

**Comment by tonyanziano** [comment]
@awalia13 thanks for the quick reply. I didn't realize at first that the second instance of the issue was also included in this bug.

**Comment by compulim** [comment]
@tonyanziano I tried few approaches, looks like this is the better one:

![image](https://user-images.githubusercontent.com/1622400/66172877-688e7380-e602-11e9-809e-c641c58c193f.png)

- Radio button is more reasonable, but
   - Radio button should be contained in a radio group
   - Radio group doesn't cooperate with `role="list"`
      - I didn't try `aria-owns` because it seems too much complexity
   - It ends up saying, "Radio button checked, 1 of 1 level 1". It can't be "1 of 5" because `role="radio"` don't play well with `role="list"`
- Button
   - `aria-pressed="true"`, Narrator didn't announce anything
   - Toggle button must be un-toggleable thru clicking, thus, we need to add uncheck logic
- Checkbox
   - `aria-checked="true"`, Narrator is announcing, only when you move the focus on it
   - It did not announce when the state changed, but I would argue this is Chromium behavior

So, I would choose `role="checkbox"` and implement as the screenshot.

See if this works for you.

**Comment by compulim** [comment]
I tried `radiogroup` and `radio`, linked them using `aria-owns`. But still, the AT said "1 of 1 level 1".

So I think the best bet is still checkbox. And there is no requirements that the user must be able to uncheck the checkbox. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

**Comment by Kumar2608** [comment]
Verified in latest bot emulator version -4.5.2-82320 with NVDA(2019.2) Now NVDA is reading the checkbox checked\Unchecked Hence we are closing this bug.
